tests/run.go

```diff
@@ -21,16 +21,24 @@ import (
 	"github.com/runfinch/common-tests/option"
 )
 
+// RunOption is the custom option to run tests in run.go.
+type RunOption struct {
+	BaseOpt    *option.Option
+	JourctlOpt *option.Option
+}
+
 // Run tests running a container image.
 // TODO: test cases for bind propagation options.
-func Run(o *option.Option) {
+func Run(o *RunOption) {
+	bo := o.BaseOpt
+	jo := o.JourctlOpt
```

Comment:
Maybe we don't this extra mapping? We'll probably [add more fields to `RunOption`](https://github.com/runfinch/common-tests/pull/5#discussion_r1038608631), and aliasing all of them seems a bit unnecessary.

```suggestion
```